Content.Server/Ghost/Components/IntrinsicRadioComponent.cs

```diff
@@ -1,53 +1,12 @@
 using Content.Server.Radio.Components;
-using Content.Server.VoiceMask;
-using Content.Shared.Chat;
-using Content.Shared.IdentityManagement;
-using Content.Shared.Radio;
-using Robust.Server.GameObjects;
-using Robust.Shared.Network;
-using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Set;
 
 namespace Content.Server.Ghost.Components
 {
     /// <summary>
-    /// Add to a particular entity to let it receive messages from the specified channels.
+    /// Add to a particular entity to let it directly receive messages. Note that this does not automatically add an <see cref="ActiveRadioComponent"/>
```

Comment:
It's good this is getting touched up too. Now might be a good time to move it out of the ghost namespace, since we've had non-ghost entities using it for a while.